packages/@ourworldindata/utils/src/Util.ts

```diff
@@ -2094,6 +2094,17 @@ export function getParentVariableIdFromChartConfig(
     return yVariableIds[0]
 }
 
+export function extractLinksFromMarkdown(markdown: string): [string, string][] {
+    const links: [string, string][] = []
+    // Captures the text inside [] and the url inside ()
+    const regex = /\[([^\]]+)\]\(([^)]+)\)/g
+    let match
+    while ((match = regex.exec(markdown))) {
```

Comment:
The while loop feels a little scary to me. Is there a reason you're not using something like `matchAll`?

Reply:
Oops! totally, good catch. This was an autocomplete that I didn't review afterwards.